platform/representation/src/gestures/DropGesture.js

```diff
@@ -165,16 +165,18 @@ define(
                     if (shouldCreateVirtualPanel(domainObject, selectedObject)){
```

Comment:
Can we pull out virtual panel creation from the drop gesture for now, seeing as they are already a temporary idea?  This behavior is a nice for functional demos but complicates integrations due to how it is hardcoded.  Would also simplify #636.